src/lib.rs

```diff
@@ -4220,7 +4244,60 @@ fn android_clang_compiler_uses_target_arg_internally(clang_path: &Path) -> bool
     false
 }
 
-fn autodetect_android_compiler(target: &str, gnu: &str, clang: &str) -> String {
+/// Returns true if `cc` has been disabled by `CC_FORCE_DISABLE`.
+pub(crate) fn is_disabled() -> bool {
```

Comment:
It's unfortunate that this is toplevel and not on `Build`, but changing everywhere that produces `Command`s to take a `&Build` was far too invasive a change. This already is pushing it.